packages/appstore/package.json

```diff
@@ -34,7 +34,6 @@
         "@deriv/trader": "^3.8.0",
         "@deriv/translations": "^1.0.0",
         "@deriv/hooks": "^1.0.0",
-        "@deriv/wallets": "^1.0.0",
```

Comment:
There is no need for dependency since we no longer serve `@deriv/wallets` in `@deriv/appstore`.